.travis.yml

```diff
@@ -32,6 +32,8 @@ _deploy_to_packagecloud: &deploy_to_packagecloud
   dist: $OS/$DIST
   package_glob: build/*.{rpm,deb}
   skip_cleanup: true
+  on:
+    all_branches: true # it's restricted with the condition for deploy stage
```

Comment:
Cron jobs should be explicitly disabled so. And also API request jobs. See [TRAVIS_EVENT_TYPE](https://docs.travis-ci.com/user/environment-variables/) variable.

Pull requests also should be excluded.

Reply:
Aren't they filtered with `if: (branch = master OR tag IS present) AND (env(PACKAGECLOUD_TOKEN) IS present)`?

Reply:
Nope. Not all at least.

Reply:
Ok. Let's switch back to 2 deploys per repo.

Reply:
I guess it would be possible to overcome the problem by checking `TRAVIS_EVENT_TYPE` in the snippet you cited. It still don't handle cron jobs and api request jobs (it'll try to deploy in them as I see); howevre we don't have them for now. PRs and commits into a fork will be handled by `env(PACKAGECLOUD_TOKEN) IS present` I guess (however I would prefer to set tarantool/queue repository for deployment explicitly).

So as I see both variants have the same behaviour and there is no reason to switch to '2 deploys per repo' way. I just proposed to make conditions a bit more explicit while we're here.

Reply:
Done.